infrastructure/main.tf

```diff
@@ -32,3 +32,18 @@ data "azurerm_key_vault" "key_vault" {
   name                = "${module.vault.key_vault_name}"
   resource_group_name = "${azurerm_resource_group.rg.name}"
 }
+
+resource "azurerm_application_insights" "appinsights" {
+  name                = "${var.product}-${var.env}"
+  location            = "${var.location}"
+  resource_group_name = "${azurerm_resource_group.rg.name}"
+  application_type    = "${var.application_type}"
+  tags                = "${var.common_tags}"
+}
+
+# store app insights key in key vault
+resource "azurerm_key_vault_secret" "bulk_scan_orchestrator_app_s2s_secret" {
+  name         = "AppInsightsInstrumentationKey"
```

Comment:
can we plz name it `app-insights-instrumentation-key` ?

Reply:
and how about jenkins pipeline using this? at first i did put in what you suggest

Reply:
that would be set in bootstrap.yaml ?